x/leverage/simulation/operations.go

```diff
@@ -312,50 +326,50 @@ func randomCoin(r *rand.Rand, coins sdk.Coins) sdk.Coin {
 // randomSpendableFields returns an random account and coins to spend in the simulation.
 func randomSpendableFields(
 	r *rand.Rand, ctx sdk.Context, accs []simtypes.Account, bk types.BankKeeper,
-) (acc simtypes.Account, spendableCoin sdk.Coin, skip bool) {
+) (acc simtypes.Account, spendableToken sdk.Coin, skip bool) {
 	acc, _ = simtypes.RandomAcc(r, accs)
 
 	spendableBalances := bk.SpendableCoins(ctx, acc.Address)
 
-	spendableCoins := simtypes.RandSubsetCoins(r, spendableBalances)
-	if spendableCoins.Empty() {
+	spendableTokens := simtypes.RandSubsetCoins(r, spendableBalances)
+	if spendableTokens.Empty() {
 		return acc, sdk.Coin{}, true
 	}
 
-	return acc, randomCoin(r, spendableCoins), false
+	return acc, randomCoin(r, spendableTokens), false
 }
 
 // randomCollateralFields returns an random account and coins to withdraw in the simulation.
 func randomCollateralFields(
 	r *rand.Rand, ctx sdk.Context, accs []simtypes.Account, lk keeper.Keeper,
-) (acc simtypes.Account, withdrawCoin sdk.Coin, skip bool) {
+) (acc simtypes.Account, withdrawToken sdk.Coin, skip bool) {
 	acc, _ = simtypes.RandomAcc(r, accs)
 
 	collateralBalances := lk.GetBorrowerCollateral(ctx, acc.Address)
 
-	withdrawCoins := simtypes.RandSubsetCoins(r, collateralBalances)
-	if withdrawCoins.Empty() {
+	withdrawTokens := simtypes.RandSubsetCoins(r, collateralBalances)
+	if withdrawTokens.Empty() {
 		return acc, sdk.Coin{}, true
 	}
 
-	return acc, randomCoin(r, withdrawCoins), false
+	return acc, randomCoin(r, withdrawTokens), false
 }
 
 // randomLiquidateFields returns two random account and coins to liquidate in the simulation.
 func randomLiquidateFields(
 	r *rand.Rand, ctx sdk.Context, accs []simtypes.Account, lk keeper.Keeper,
-) (liquidator simtypes.Account, borrower simtypes.Account, repaymentCoin sdk.Coin, skip bool) {
+) (liquidator simtypes.Account, borrower simtypes.Account, repaymentToken sdk.Coin, skip bool) {
 	idxLiquidator := r.Intn(len(accs) - 1)
 
 	liquidator = accs[idxLiquidator]
 	borrower = accs[idxLiquidator+1]
 
 	collateralBalances := lk.GetBorrowerCollateral(ctx, borrower.Address)
 
-	repaymentCoins := simtypes.RandSubsetCoins(r, collateralBalances)
-	if repaymentCoins.Empty() {
+	repaymentTokens := simtypes.RandSubsetCoins(r, collateralBalances)
```

Comment:
Since these are collateral `uTokens` and not borrowed `tokens`, consider calling them `rewardTokens` instead of `repaymentTokens`. Liquidate repays borrowed `tokens` and receives collateral `uTokens` as a reward.

Reply:
Additional note, after seeing how it's used:

it looks like this function needs one more return - specifically, liquidation requires a `repaymentTokens` from `GetBorrowerBorrows(borrower.Address)` and a `rewardDenom` from `GetBorrowerCollateral(borrower.Address)`.